packages/contracts/scripts/deploy/dollar/solidityScripting/03_UbiquityGovernanceToken.s.sol

```diff
@@ -4,18 +4,33 @@ pragma solidity 0.8.19;
 import "./02_UbiquityDollarToken.s.sol";
 
 contract GovernanceScript is DollarScript {
-    UbiquityGovernanceToken governance;
+    UbiquityGovernanceToken public uGovToken;
```

Comment:
We have a convention to omit all token symbols like `UAD, UCR, UGOV, etc...`

Whenever possible they should be referenced like `dollarToken`, `creditToken`, `creditNft`, `governanceToken`, etc...

Pls refactor accordingly